digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/service/pointasset/TrafficSignService.scala

```diff
@@ -238,6 +242,28 @@ class TrafficSignService(val roadLinkService: RoadLinkService, val userProvider:
     }
   }
 
+  override def getChanged(sinceDate: DateTime, untilDate: DateTime): Seq[ChangedPointAsset] = { throw new UnsupportedOperationException("Not Supported Method, Try to used") }
```

Comment:
This method should be replace by 
**getChanged( sinceDate: DateTime, untilDate: DateTime, trafficSignTypes: Set[Int] = Set())** on base and could be remove the Unsupported exception